build_all.bat

```diff
@@ -215,5 +216,13 @@ call group5\build.bat 2019.4.2f1
 call group5\build.bat 2019.4.3f1
 call group5\build.bat 2019.4.4f1
 call group5\build.bat 2019.4.5f1
+call group5\build.bat 2019.4.6f1
+call group5\build.bat 2019.4.7f1
+call group5\build.bat 2019.4.8f1
+call group5\build.bad 2019.4.9f1
```

Comment:
typo, should be `.bat` (surprised this doesn't throw a fatal error - maybe we should add some sanity checking)

Reply:
It did... I noticed right before I went to bed as I'd left it running in another window. Fixed.